roles/self_hosted_first_host/defaults/main.yml

```diff
@@ -10,7 +10,7 @@ interface_name: '{{ ansible_default_ipv4.interface }}'
 cluster_name: Default
 storage_datacenter_name: hosted_storage
 hosted_storage_type: nfs
-storage_domain_type: "{{ 'nfs3' if hosted_storage_type == 'nfs' else 'glusterfs' if hosted_storage_type == 'gluster' else hosted_storage_type }}"
+storage_domain_type: "{{ 'nfs3' if hosted_storage_type == 'nfs' else 'glusterfs' if hosted_storage_type == 'gluster' else 'fc' if hosted_storage_type == 'fcp' else hosted_storage_type }}"
```

Comment:
This is getting a bit unwieldy, I wonder if there's a better way we can handle this. Not a blocker for merge, just a thought.